source/bsondump-installation.txt

```diff
@@ -0,0 +1,31 @@
+.. _bsondump-installation:
+
+=========================
+``bsondump`` Installation
+=========================
+
```

Comment:
I think you can omit this as a single page, as it doesn't really add much. What do you think about either including this short install section on the compatibility page OR making a single install page that covers all the tools. I'm concerned that breaking out the install content this way will result in a dozen identical pages.